common/unity_install.sh

```diff
@@ -8,12 +8,7 @@ patch_xml() {
     VALC="value"; VAL="$4"
   fi
   case $2 in
-    *dialer_phenotype_flags*.xml)  sed -i "/#DIALERPATCHES/a\          patch_xml $1 \$MODPATH/\ '$3' \"$4\"" $INSTALLER/common/post-fs-data.sh; VAR1=boolean; VAR2=string;;
-    *mixer_paths*.xml) sed -i "/#MIXERPATCHES/a\                       patch_xml $1 \$MODPATH/\$NAME '$3' \"$4\"" $INSTALLER/common/aml.sh; VAR1=ctl; VAR2=mixer;;
```

Comment:
We should undo this somehow

Reply:
if [ $API -ge 28 ]; then
  ui_print " "
  ui_print "   Disabling Google's Call Screening..."
  DPF=$(find /data/data/com.google.android.dialer* -name "dialer_phenotype_flags.xml")
  if [ -f $DPF ]; then
    sed -i "/<!--$MODID-->/d" $DPF
    sed -i -e "s|<!--$MODID\(.*\)|\1|g" -e "s|\(.*\)$MODID-->|\1|g" $DPF
    if $BOOTMODE; then
      am force-stop "com.google.android.dialer"
    fi
  fi
fi

this uninstalls it. since we add the $MODID to the end of the patch we just search for the MODID with sed and remove